src/button-dropdown/styles.scss

```diff
@@ -70,6 +70,14 @@ $dropdown-trigger-icon-offset: 2px;
       border-end-end-radius: 0;
       padding-inline-end: awsui.$space-m;
       margin-inline-end: awsui.$space-xxxs;
+
+      &.has-no-text {
+        padding-inline: awsui.$space-button-icon-only-horizontal;
+
+        &.visual-refresh {
+          padding-inline-start: calc(#{awsui.$space-s} - #{$dropdown-trigger-icon-offset});
+        }
```

Comment:
Why is this spacing difference is not built into the design token by default?

Reply:
good point, I just reused the existing value